examples/ios/swift-2.2/RealmExamples.xcodeproj/project.pbxproj

```diff
@@ -1089,6 +1141,56 @@
 			};
 			name = Release;
 		};
+		E8F14D0F1CF8FD0C00564AF5 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				CLANG_ANALYZER_NONNULL = YES;
```

Comment:
A lot of these settings are redundant and repeat those set at the project level. Only the handful that have different values than the project level need to be set on the target.